chainer/functions/matmul.py

```diff
@@ -0,0 +1,210 @@
+import ctypes
+import numpy
```

Comment:
Add a blank line between imports for standard libraries and third-party libraries.